### ISSUE TITLE
Please add generic optocouplers 

### ISSUE THREAD

**Issue by papilinou** [circuitdiagram/components#101]
A standard electronic component is missing : optocoupler.
Please add it in 4,6,8 pins.

**Comment by Corsaka** [comment]
Can't you just combine an LED and a phototransistor?

**Comment by papilinou** [comment]
yes,that's what i do at each time i need one.I'm now studying component design and near to set a pull request.I'll test it today on CD desktop.
The only thing not clear is how to get an GUID when using gitpod.

**Comment by SamuelFisher** [comment]
> The only thing not clear is how to get an GUID when using gitpod.

You can create a GUID by running this command in the terminal in Gitpod:

```bash
cat /proc/sys/kernel/random/uuid
```

**Comment by papilinou** [comment]
thanks,done.component tested in CD desktop and fine but no icon.Can't find example how to add.

**Comment by papilinou** [comment]
set a pull request for optocoupler.
